maptivist/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -55,22 +55,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
  
  
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        String title = request.getParameter("title");
-        String longitude = request.getParameter("longitude");
-        String latitude = request.getParameter("latitude");
-        String description = request.getParameter("description"); 
-        Set<String> linkSet = new HashSet<String>(Arrays.asList(request.getParameterValues("links")));
+        String email = request.getParameter("email");
+        String title = request.getParameter("marker-title");
+        String longitude = request.getParameter("marker-lng");
+        String latitude = request.getParameter("marker-lat");
+        String description = request.getParameter("marker-description"); 
+        Set<String> linkSet = new HashSet<String>(Arrays.asList(request.getParameter("marker-link")));
         String links = createLinkString(linkSet);
-        Set<String> categorySet = new HashSet<String>(Arrays.asList(request.getParameterValues("category")));
+        Set<String> categorySet = new HashSet<String>(Arrays.asList(request.getParameter("marker-category")));
         String categories = createCategoriesString(categorySet);
         String flag = request.getParameter("flags");
-        Boolean voteCheck = Boolean.parseBoolean(request.getParameter("upvotes"));
+        Boolean voteCheck = true; // Boolean.parseBoolean(request.getParameter("votes"));
```

Comment:
Did you mean to set this to true always?

That means the
```
if(!voteCheck) {}
```
block below is now dead code. Should that be deleted?

Reply:
Yes for now. The VoteFunctionality PR makes changes to that. It is assumed that the creator of the marker would vote for it

Reply:
Ok - please add a comment to the code explaining why voteCheck is always true, and any future changes you expect to make to the code.